assets/src/apps/delivery/store/features/groups/actions/deck.ts

```diff
@@ -56,7 +56,7 @@ export const initializeActivity = createAsyncThunk(
     const currentSequenceId = sequence.find((entry) => entry.activity_id === activityId)?.custom
       .sequenceId;
     if (!currentSequenceId) {
-      throw new Error(`Activity ${activityId} not found in sequence!`);
+      throw new Error(`deck::initializeActivity - Activity ${activityId} not found in sequence!`);
```

Comment:
I was trying to track down a `Activity ${activityId} not found in sequence` console log, but there were several of them, so I added a bit of context